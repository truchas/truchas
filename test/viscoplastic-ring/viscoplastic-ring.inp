-----------------
Ring-Plug Assembly Viscoplastic Residual Stress Calculation, MTS version


================================================================================

David Korzekwa, MST6

================================================================================

*All units are SI, with temperatures in degrees K
*Mesh file coordinates are in meters
*This mesh contains two materials
	#1 Graphite plug with elastic properties (no temperature dependence)
	#2 5754 Aluminum ring with elastic-viscoplastic properties

This problem uses a coarse quarter symmetry mesh. The problem starts with both
materials below their stress reference temperature.  The initial stress is
calculated and a small amount of plastic relaxation occurs.  If the total time is
increased, the assembly will continue to cool and the elastic stress and plastic
strain will continue to evolve.  This version for the test suite runs for a
short time, exercising the initial elastic stress calculation, viscoplastic flow
and heat conduction.  This version uses the MTS model for the viscoplastic model

define a three-dimensional mesh

&MESH

    mesh_file         = 'hytec_quarter_coarse_g.exo', 
    gap_element_blocks = 3
/

define the plug geometry and initial conditions

&BODY

    material_number       = 1, 
    mesh_material_number  = 1, 
    surface_name          = 'from mesh file', 
    temperature           = 5.50e+02, 

/
  
define the ring geometry and initial conditions

&BODY

    material_number       = 2, 
    mesh_material_number  = 2, 
    surface_name          = 'from mesh file', 
    temperature           = 5.50e+02, 

/
Gap block
&BODY

    material_number       = 3, 
    mesh_material_number  = 3, 
    surface_name          = 'from mesh file', 
    temperature           = 5.50e+02, 

/


define the boundary conditions:

z = 0.0 insulated with zero z displacement (symmetry plane)

&DS_BOUNDARY_CONDITION

    name           = 'bottom surface'
    variable       = 'temperature'
    condition      = 'flux'
    data_constant  = 0.0
    face_set_ids   = 6

/

&BC

    surface_name   = 'from mesh file'
    mesh_surface   = 6, 
    bc_variable    = 'displacement', 
    bc_type        = 'z-displacement', 
    bc_value       = 0.0e0, 

/

z = 0.0127 heat transfer and all traction components zero (default)

&DS_BOUNDARY_CONDITION
    name          = 'top surface'
    variable      = 'temperature'
    condition     = 'HTC'
    data_constant = 1.0e2, 2.98e+02
    face_set_ids  = 5
/

x = 0.0 insulated with zero x displacement (symmetry plane)

&DS_BOUNDARY_CONDITION
    name          = 'x=0 symmetry plane'
    variable      = 'temperature'
    condition     = 'flux'
    data_constant = 0.0
    face_set_ids  = 3
/

&BC

    surface_name   = 'from mesh file'
    mesh_surface   = 3, 
    bc_variable    = 'displacement', 
    bc_type        = 'x-displacement', 
    bc_value       = 0.0e0, 

/

y = 0.0 insulated with zero y displacement (symmetry plane)

&DS_BOUNDARY_CONDITION

    name = 'y=0 symmetry plane'
    variable = 'temperature'
    condition = 'flux'
    data_constant = 0.0
    face_set_ids  = 2

/

&BC

    surface_name   = 'from mesh file'
    mesh_surface   = 2, 
    bc_variable    = 'displacement', 
    bc_type        = 'y-displacement', 
    bc_value       = 0.0e0, 

/

Outer radius = 0.0508 heat transfer and all tractions zero

&DS_BOUNDARY_CONDITION

    name           = 'outer radius'
    variable       = 'temperature'
    condition      = 'HTC'
    data_constant  = 1.0e2, 2.98e+02
    face_set_ids   = 4

/

Interface between plug and ring

&DS_INTERFACE_CONDITION

    name           = 'plug/ring interface'
    variable       = 'temperature'
    condition      = 'HTC'
    data_constant  = 1.0e3
    face_set_ids   = 3
 
/

&BC

    surface_name   = 'from mesh file'
    mesh_surface   = 1, 
    bc_variable    = 'displacement', 
    bc_type        = 'normal-constraint', 
    bc_value       = 0.0e0, 

/

define the plug material properties
 
&MATERIAL

    Immobile                      = .true.
    material_number               = 1, 
    priority                      = 1, 
    material_name                 = 'graphite', 
    density                       = 1.7e+03, 
    Material_Feature              = 'background'

/

&PHASE

   name = 'graphite'
   property_name(1) = 'density',              property_constant(1) = 1.7e3
   property_name(2) = 'specific heat',        property_constant(2) = 1.925e3
   property_name(3) = 'conductivity',         property_constant(3) = 1.95e2
   property_name(4) = 'TM reference density',     property_constant(4) = 1.7e3
   property_name(5) = 'TM reference temperature', property_constant(5) = 800.0 
   property_name(6) = 'TM linear CTE',            property_constant(6) = 7.0e-6
   property_name(7) = 'Lame1', property_constant(7) = 3.4e+9
   property_name(8) = 'Lame2', property_constant(8) = 2.76e+9

/

&MATERIAL_SYSTEM

   name = 'graphite'
   phases = 'graphite'

/

define the ring material properties
 
&MATERIAL

    Immobile                      = .true.
    material_number               = 2
    priority                      = 2
    material_name                 = '5754 aluminum' 
    density                       = 2.70e+03 

/

&VISCOPLASTIC_MODEL
    phase = '5754 aluminum'
    model = 'MTS'
    MTS_k       = 1.38e-23
    MTS_mu_0    = 28.815e9
    MTS_sig_a   = 10.0e6
    MTS_d       = 3.440e9
    MTS_temp_0  = 215.0
    MTS_b       = 2.86e-10
    MTS_edot_0i = 1.0e7
    MTS_g_0i    = 3.6
    MTS_q_i     = 1.5
    MTS_p_i     = 0.5
    MTS_sig_i   = 107.2e6
/

&PHASE

    name = '5754 aluminum'
    property_name(1) = 'density',              property_constant(1) = 2.7e3
    property_name(2) = 'specific heat',        property_constant(2) = 8.5e2
    property_name(3) = 'conductivity',         property_constant(3) = 2.4e2
    property_name(4) = 'TM reference density',     property_constant(4) = 1.7e3
    property_name(5) = 'TM reference temperature', property_constant(5) = 800.0 
    property_name(6) = 'TM linear CTE',            property_constant(6) = 2.2e-5
    property_name(7) = 'Lame1', property_constant(7) = 5.20e+10
    property_name(8) = 'Lame2', property_constant(8) = 2.60e+10

/

&MATERIAL_SYSTEM

    name = '5754 aluminum'
    phases = '5754 aluminum'

/

Gap material
 
&MATERIAL

    Immobile                      = .true.
    material_number               = 3, 
    priority                      = 3, 
    material_name                 = 'gap', 
    density                       = 0.0,
/

solid mechanics and heat transfer

&PHYSICS

    fluid_flow       = .false., 
    heat_transport  = .true., 
    solid_mechanics  = .true., 

/

&DIFFUSION_SOLVER

    abs_temp_tol       = 0.0
    rel_temp_tol       = 1.0e-4
    abs_enthalpy_tol   = 0.0
    rel_enthalpy_tol   = 1.0e-4
    nlk_tol            = 0.05
    nlk_preconditioner = 'hypre_amg'
    verbose_stepping   = .true.

/

&LINEAR_SOLVER

    name                    = 'displacement precon', 
    method                  = 'none', 
    preconditioning_steps   = 4, 
    relaxation_parameter    = 1.6, 
    preconditioning_method  = 'tm_ssor', 
/

&NONLINEAR_SOLVER 

    name                    = 'displacement nonlin', 
    method                  = 'nlk', 
    linear_solver_name      = 'displacement precon', 
    convergence_criterion   = 1e-13, 
    nlk_max_vectors         = 40
    nlk_vector_tolerance    = 0.01
    maximum_iterations      = 1000
/

Transient heat transfer and elastic_viscoplasticity

&NUMERICS

    dt_constant = 1.0e-1
    dt_max                           = 1.0e0, 
    dt_grow                          = 5.0,
    dt_init                          = 1.0e-1, 
    dt_min                           = 1.0e-8, 
    strain_limit                     = 1.0e-10, 
    displacement_nonlinear_solution  = 'displacement nonlin', 

/

&OUTPUTS

    output_t                    = 0.0, 0.5 !5.0 
    output_dt                   = 1.0
    !short_output_dt_multiplier  = 1, 

/

&PARALLEL_PARAMETERS

    Partitioner  = 'automatic', 

/
