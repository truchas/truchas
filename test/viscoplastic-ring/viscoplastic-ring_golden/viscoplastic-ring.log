
================================================================================

PROGRAM INFORMATION

   code:                DUMMY
   version:             2.9.dev
   libraries:           UbikSolve, PGSLib, Chaco
   build architecture:  x86_64
   build date/time:     27 Jun 2015 00:59:08
   build flags:         -O -u -fPIC-O3
   build host:          noether.lanl.gov
   run architecture:    Linux noether.lanl.gov 3.15.10-201.fc20.x86_64 #1 SMP Wed Aug 27 21:10:06 UTC 2014 x86_64
   run host:            noether.lanl.gov
   run date/time:       27 Jun 15 22:41:44
   processors:          1

================================================================================

COPYRIGHT

   Copyright 2007-2013. Los Alamos National Security, LLC.

   This material was produced under U.S. Government contract DE-AC52-06NA25396
   for Los Alamos National Laboratory (LANL), which is operated by Los Alamos
   National Security, LLC for the U.S. Department of Energy. The U.S. Government
   has rights to use, reproduce, and distribute this software.  NEITHER THE
   GOVERNMENT NOR LOS ALAMOS NATIONAL SECURITY, LLC MAKES ANY WARRANTY, EXPRESS
   OR IMPLIED, OR ASSUMES ANY LIABILITY FOR THE USE OF THIS SOFTWARE. If software
   is modified to produce derivative works, such modified software should be
   clearly marked, so as not to confuse it with the version available from LANL.

   Additionally, this program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License as published by
   the Free Software Foundation; either version 2 of the License, or (at your
   option) any later version. Accordingly, this program is distributed in the
   hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
   implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
   See the GNU General Public License for more details.

================================================================================

DISCLAIMER

   This Truchas release is registered with the Los Alamos National
   Laboratory (LANL) as Los Alamos Computer Code LA-CC-07-031.
DANU: Opening h5 output file

================================================================================

INPUT

Opening input file viscoplastic-ring.inp ...
 Reading PARALLEL_PARAMETERS Namelist ...

Reading FUNCTION namelists ...

Reading PHASE namelists ...
  Reading PHASE namelist #1
  Reading PHASE namelist #2

Reading MATERIAL_SYSTEM namelists ...
  Reading MATERIAL_SYSTEM namelist #1
    using default REFERENCE_TEMP =  0.000E+00
    using default REFERENCE_ENTHALPY =  0.000E+00
  Reading MATERIAL_SYSTEM namelist #2
    using default REFERENCE_TEMP =  0.000E+00
    using default REFERENCE_ENTHALPY =  0.000E+00

Reading PHYSICS namelist ...

 Reading OUTPUTS Namelist ...

Reading MESH Namelist ...
         Opened mesh file ./hytec_quarter_coarse_g.exo
           containing 496 nodes and 315 cells

 Reading MATERIAL Namelist # 1 ...

 Reading MATERIAL Namelist # 2 ...

 Reading MATERIAL Namelist # 3 ...
This is the last MATERIAL Namelist.
Identified 3 material(s)

               Material Priorities

         Material     Name     Priority
         --------     ----     --------
             1     graphite        1
             2     5754 alu        2
             3          gap        3

Reading VISCOPLASTIC_MODEL namelists ...
  Reading VISCOPLASTIC_MODEL namelist #1

Reading INTERFACES namelist ...
  INTERFACES namelist not found; using defaults

 Reading BODY Namelist # 1 ...

                                   Geometry Data

          Body  Material  Surface  Translation  Rotation  Rotation  Surface
                 Number     Type      Point       Point     Angle  Parameters
          ----  --------  -------  -----------  --------  -------- ----------
            1       1    from mes   0.000E+00   0.000E+00    0.0    0.000E+00
                         (none   )  0.000E+00   0.000E+00    0.0    0.000E+00
                                    0.000E+00   0.000E+00    0.0    0.000E+00

 Reading BODY Namelist # 2 ...
            2       2    from mes   0.000E+00   0.000E+00    0.0    0.000E+00
                         (none   )  0.000E+00   0.000E+00    0.0    0.000E+00
                                    0.000E+00   0.000E+00    0.0    0.000E+00

 Reading BODY Namelist # 3 ...
            3       3    from mes   0.000E+00   0.000E+00    0.0    0.000E+00
                         (none   )  0.000E+00   0.000E+00    0.0    0.000E+00
                                    0.000E+00   0.000E+00    0.0    0.000E+00
         BODY Namelist number  1 will be used for background material ( 1)

 Reading LINEAR_SOLVER Namelist(s) ...

         Reading LINEAR_SOLVER Namelist # 1 ...

Reading NONLINEAR_SOLVER Namelists ...
  Reading NONLINEAR_SOLVER Namelist #1

 Reading NUMERICS Namelist ...
Using default linear solver parameters for displacement linear solution.
Using nonlinear solver "displacement nonlin" for displacement nonlinear solution.
Using linear solver "displacement precon" for nonlinear solver "displacement nonlin".

 Reading BC Namelists ...

         Reading BC namelist #  1: Unnamed

         Reading BC namelist #  2: Unnamed

         Reading BC namelist #  3: Unnamed

         Reading BC namelist #  4: Unnamed

Reading DIFFUSION_SOLVER namelist ...
  using default STEPPING_METHOD value: "Adaptive BDF2"
  using default MAX_NLK_ITR value: 5
  using default MAX_STEP_TRIES value: 10
  using default MAX_NLK_VEC value: MAX_NLK_ITR - 1
  using default NLK_VEC_TOL value: 1.00E-03
  WARNING: using a pure relative error norm; temp must be bounded away from 0
  WARNING: using a pure relative error norm; enthalpy must be bounded away from 0
  using default PC_AMG_CYCLES value: 1

Reading DS_BOUNDARY_CONDITION namelists ...
  [bottom surface] "flux" condition for "temperature" variable
  [top surface] "HTC" condition for "temperature" variable
  [x=0 symmetry plane] "flux" condition for "temperature" variable
  [y=0 symmetry plane] "flux" condition for "temperature" variable
  [outer radius] "HTC" condition for "temperature" variable

Reading DS_INTERFACE_CONDITION namelists ...
  [plug/ring interface] read "HTC" condition for "temperature" variable

Reading DS_SOURCE namelists ...
  No DS_SOURCE namelists found.

Reading ENCLOSURE_RADIATION namelists ...

Reading ENCLOSURE_SURFACE namelists ...

 Reading PROBE namelists ...
         This is the last PROBE Namelist.

         Identified 0 probe(s)

Input file viscoplastic-ring.inp closed.

================================================================================

INITIALIZATION

Allocating mesh derived types ... done.
 Automatic partitioner selection: None

Reading ExodusII mesh file ./hytec_quarter_coarse_g.exo ...
 Closed ExodusII mesh file ./hytec_quarter_coarse_g.exo

 Determining mesh partitions and permutation vectors.

 Permuting mesh ... done.
 Permuting vertices ... done.
 Renumbering vertices ... done.

 Establishing mesh connectivity for      1890 faces ...

   Sorting face vertex 1 ...
   Sorting face vertex 2 ...
   Sorting face vertex 3 ...
   Sorting face vertex 4 ...
   There are        318 external boundary faces.
 Mesh connectivity established.

 Finding all cell neighbors ... done.

Allocating base derived types ...done.

                               Mesh Diagnostics
                               ----------------

                   Element Type   Number     Face Type   Number
                   ------------   ------     ---------   ------
                      Hex           315       Point           0
                      Prism           0       Line            0
                      Pyramid         0       Triangle        0
                      Tet             0       Quad         1890

                            Nodes               Cells
                            -----               -----
                                6 are shared by   1
                               72 are shared by   2
                                2 are shared by   3
                              240 are shared by   4
                                2 are shared by   6
                              174 are shared by   8

Computing cell geometry ... 
                    Min        Cell       Max        Cell       Total
                    ---        ----       ---        ----       -----
       Volumes   4.4409E-16       286   1.3648E-07       119    2.5635E-05


              210 out of      315 total cells are orthogonal

    use_ortho_face_gradient = F

    Using full pivoting for LSLR_ operators

done.

                    Min Coord        Max Coord
                    ---------        ---------
                   0.0000E+00       5.0800E-02
                   0.0000E+00       5.0800E-02
                   0.0000E+00       1.2700E-02

 Finding face neighbors ... done.

Initializing distributed mesh MAIN ...
  Reading ExodusII mesh file ./hytec_quarter_coarse_g.exo
  UNSTR_MESH Profile:
     PE|    nnode    nface    ncell
    ---+---------------------------
      1|      496     1031      285
  Mesh Communication Profile:
              Nodes           Faces           Cells
     PE|  off-PE   on-PE  off-PE   on-PE  off-PE   on-PE
    ---+------------------------------------------------
      1|      0      496      0     1031      0      285
  Distributed mesh MAIN initialized.
DANU: adding default mesh entry
DANU: writing mesh node coordinates
DANU: adding main simulation entry

Computing initial volume fractions ...

    Volume fractions will be computed in        315 total cells with   5**3 particles/cell

    Tallying        5**3 =    125 interface particles

    Tallied   1 x   5**2 =     25 interface particles
    Tallied   2 x   5**2 =     50 interface particles
    Tallied   3 x   5**2 =     75 interface particles
    Tallied   4 x   5**2 =    100 interface particles
    Tallied   5 x   5**2 =    125 interface particles
ERROR: volume fraction sum < 1 in cells: 1 2 6 8 9 [119 more items omitted]
       minimum volume fraction sum less 1: -3.44169E-15
  Computed volume fractions are invalid; attempting to normalize.
  Normalization was successful.
  Initial volume fractions computed.
 Initializing Displacement Boundary Conditions

 Locating cell faces for which BCs are to be applied ...




Warning:     15 interior #2 faces are affected by BC namelist  4!
Warning:     15 interior #4 faces are affected by BC namelist  4!
Warning:     30 interior #5 faces are affected by BC namelist  4!
Warning:     60 interior #6 faces are affected by BC namelist  4!

    Boundary conditions in BC namelist  1 will be applied on an area of   2.01851E-03 (    95 faces)
    Boundary conditions in BC namelist  2 will be applied on an area of   6.45160E-04 (    36 faces)
    Boundary conditions in BC namelist  3 will be applied on an area of   6.45160E-04 (    36 faces)
    Boundary conditions in BC namelist  4 will be applied on an area of   2.02475E-03 (   120 faces)
 Initializing Pressure Boundary Conditions
   DIRICHLET: 0 boundary points
   REFLECTIVE: 318 boundary points
   EXTERIOR: 318 boundary points
   NEUMANN: 0 boundary points
   HNEUMANN: 318 boundary points
 Pressure BCs initialized.

 Finished BC initialization.

 Finding control volume faces internal to cells ...

 Finding jacobians ...

 Inverting jacobians ...
done.

 Finding control volume faces on external boundaries ...

done.

 Calculating initial thermo-elastic state. Done.

     0 Thermo-elastic iterations (linear)
     124 Thermo-elastic iterations (nonlinear)

Initializing diffusion solver ...
  Generating external source for "temperature" equation
  Generating "HTC" interface condition for "temperature" variable
    using DS_INTERFACE_CONDITION[1]: plug/ring interface
  Generating "radiation" interface condition for "temperature" variable
  Generating "HTC" boundary condition for "temperature" variable
    using DS_BOUNDARY_CONDITION[2]: top surface
    using DS_BOUNDARY_CONDITION[5]: outer radius
  Generating "radiation" boundary condition for "temperature" variable
  Generating "dirichlet" boundary condition for "temperature" variable
  Generating "flux" boundary condition for "temperature" variable
    using DS_BOUNDARY_CONDITION[1]: bottom surface
    using DS_BOUNDARY_CONDITION[3]: x=0 symmetry plane
    using DS_BOUNDARY_CONDITION[4]: y=0 symmetry plane
  Diffusion solver initialized.

Computing consistent initial state for HT/SD solver ...
  Computing consistent face temperatures and radiosities ...
    ||Rface(4)||= 6.16E-06, ||Rface(0)||= 7.18E+00, ||ΔTface||_max= 1.89E-05

Computing consistent initial state derivative for HT/SD solver ...
  Computing consistent face temperatures and radiosities ...
    ||Rface(4)||= 1.82E-05, ||Rface(0)||= 1.05E+01, ||ΔTface||_max= 7.72E-05

================================================================================

EXECUTION


          1: t =   0.00000E+00, dt(constant) =   1.00000E-01

DS: dt=1.000E-01, NFUN:NPC=0000002:00001, NNR:NNF:NSR=0000:0000:0000
            0 Solid mechanics iterations (linear)
          106 Solid mechanics iterations (nonlinear)

          2: t =   1.00000E-01, dt(constant) =   1.00000E-01

DS: dt=1.000E-01, NFUN:NPC=0000005:00001, NNR:NNF:NSR=0000:0000:0000
            0 Solid mechanics iterations (linear)
          102 Solid mechanics iterations (nonlinear)

          3: t =   2.00000E-01, dt(constant) =   1.00000E-01

DS: dt=1.000E-01, NFUN:NPC=0000008:00001, NNR:NNF:NSR=0000:0000:0000
            0 Solid mechanics iterations (linear)
          104 Solid mechanics iterations (nonlinear)

          4: t =   3.00000E-01, dt(constant) =   1.00000E-01

DS: dt=1.000E-01, NFUN:NPC=0000011:00001, NNR:NNF:NSR=0000:0000:0000
            0 Solid mechanics iterations (linear)
          104 Solid mechanics iterations (nonlinear)

          5: t =   4.00000E-01, dt(constant) =   1.00000E-01

DS: dt=1.000E-01, NFUN:NPC=0000014:00001, NNR:NNF:NSR=0000:0000:0000
            0 Solid mechanics iterations (linear)
          105 Solid mechanics iterations (nonlinear)

================================================================================

TERMINATION

DANU: Closing h5 output file
Warning: DANU: Danu fid is still associated
                 Final Time:  5.0000E-01 after     5 steps


  TIMING SUMMARY                       AVERAGE        MIN          MAX
  --------------                      ---------    ---------    ---------
  Total                               3.236E+00    3.236E+00    3.236E+00
    Input                               9.000E-03    9.000E-03    9.000E-03
    Initialization                      3.020E-01    3.020E-01    3.020E-01
      Solid Mechanics                     2.570E-01    2.570E-01    2.570E-01
        Precondition                        0.000E+00    0.000E+00    0.000E+00
        NK                                  2.550E-01    2.550E-01    2.550E-01
          NK Residual                         1.000E-03    1.000E-03    1.000E-03
          Newton Loop                         2.540E-01    2.540E-01    2.540E-01
            NK LS                               8.400E-02    8.400E-02    8.400E-02
              Solver TMP1                         8.300E-02    8.300E-02    8.300E-02
                Precondition                        8.300E-02    8.300E-02    8.300E-02
            NK Residual                         1.590E-01    1.590E-01    1.590E-01
      Initialization                      6.000E-03    6.000E-03    6.000E-03
        Solid Mech Init                     6.000E-03    6.000E-03    6.000E-03
      HTSD function                       0.000E+00    0.000E+00    0.000E+00
        HT function                         0.000E+00    0.000E+00    0.000E+00
      ssor-setup                          0.000E+00    0.000E+00    0.000E+00
      ssor-solve                          0.000E+00    0.000E+00    0.000E+00
    Main Cycle                          2.923E+00    2.923E+00    2.923E+00
      Output                              8.000E-03    8.000E-03    8.000E-03
      Time Step                           0.000E+00    0.000E+00    0.000E+00
      Diffusion Solver                    4.000E-03    4.000E-03    4.000E-03
        HTSD precon compute                 2.000E-03    2.000E-03    2.000E-03
          hypre-matrix-copy                   0.000E+00    0.000E+00    0.000E+00
          boomer-setup                        1.000E-03    1.000E-03    1.000E-03
        HTSD function                       2.000E-03    2.000E-03    2.000E-03
          HT function                         2.000E-03    2.000E-03    2.000E-03
        HTSD precon apply                   0.000E+00    0.000E+00    0.000E+00
          HT precon apply                     0.000E+00    0.000E+00    0.000E+00
            boomer-solve                        0.000E+00    0.000E+00    0.000E+00
      Solid Mechanics                     2.910E+00    2.910E+00    2.910E+00
        Precondition                        0.000E+00    0.000E+00    0.000E+00
        NK                                  2.902E+00    2.902E+00    2.902E+00
          NK Residual                         2.200E-02    2.200E-02    2.200E-02
          Newton Loop                         2.880E+00    2.880E+00    2.880E+00
            NK LS                               3.770E-01    3.770E-01    3.770E-01
              Solver TMP1                         3.770E-01    3.770E-01    3.770E-01
                Precondition                        3.770E-01    3.770E-01    3.770E-01
            NK Residual                         2.445E+00    2.445E+00    2.445E+00

                  Process virtual memory used: 8.46E+01 mB
                                   words/cell: 35191

truchas terminated normally on 27 Jun 15 at 22:41:47
