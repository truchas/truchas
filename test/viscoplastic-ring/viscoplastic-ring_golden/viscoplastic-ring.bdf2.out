
TRAP step      1: T= 0.00000E+00, H= 1.00000E-01, ETAH= 5.00000E-02
  Preconditioner updated at T= 1.00000E-01, ETAH= 5.00000E-02
    1: error= 1.06390E+00
    2: error= 3.35971E-02
  AIN BCE solve succeeded:   2 iterations, error= 3.35971E-02
  Step accepted: no local error control

BDF2 step      2: T= 1.00000E-01, H= 1.00000E-01, ETAH= 6.66667E-02
    1: error= 4.87463E-01
    2: error= 1.27901E-01
    3: error= 2.36419E-03
  AIN BCE solve succeeded:   3 iterations, error= 2.36419E-03
  Step accepted: no local error control

BDF2 step      3: T= 2.00000E-01, H= 1.00000E-01, ETAH= 6.66667E-02
    1: error= 4.46382E-01
    2: error= 1.18673E-01
    3: error= 1.98453E-03
  AIN BCE solve succeeded:   3 iterations, error= 1.98453E-03
  Step accepted: no local error control

BDF2 step      4: T= 3.00000E-01, H= 1.00000E-01, ETAH= 6.66667E-02
    1: error= 3.02858E-01
    2: error= 8.16585E-02
    3: error= 1.27332E-03
  AIN BCE solve succeeded:   3 iterations, error= 1.27332E-03
  Step accepted: perr= 3.77117E-01

BDF2 step      5: T= 4.00000E-01, H= 1.00000E-01, ETAH= 6.66667E-02
    1: error= 1.83364E-01
    2: error= 5.03691E-02
    3: error= 7.59995E-04
  AIN BCE solve succeeded:   3 iterations, error= 7.59995E-04
  Step accepted: perr= 2.29333E-01
